c10/core/TensorImpl.cpp

```diff
@@ -880,8 +872,16 @@ void TensorImpl::set_sym_sizes_and_strides(
     c10::SymIntArrayRef strides) {
   has_symbolic_sizes_strides_ = true;
   sizes_strides_policy_ = static_cast<uint8_t>(SizesStridesPolicy::CustomSizes);
-  sizes_and_strides_.set_sizes(sizes);
-  sizes_and_strides_.set_strides(strides);
+  if (!extra_meta_)
+    extra_meta_ = std::make_unique<ExtraMeta>();
```

Comment:
please please please use braces

Reply:
I can't, this is clang-format ordained >_<